blazar/manager/enforcement.py

```diff
@@ -234,27 +238,47 @@ def _get_project_name(self, project_id):
         project = self.keystone_client.projects.get(project_id)
         return project.name
 
-    def get_balance(self, project_name):
+    def _get_project_enforcement_id(self, project_id):
+        """Get project name or charge_code from Keystone"""
+        self.keystone_client = keystone.BlazarKeystoneClient(
+            username=CONF.os_admin_username,
+            password=CONF.os_admin_password,
+            tenant_name=CONF.os_admin_project_name)
+        project = self.keystone_client.projects.get(project_id)
+        try:
+            enforcement_attribute = getattr(project,CONF.enforcement.project_enforcement_id)
+            LOG.info('Enforcement_attribute: '+CONF.enforcement.project_enforcement_id+'. Project: '+enforcement_attribute)
+            if not enforcement_attribute:
+                msg = 'Enforcement attribute project_enforcement_id {} is null in keystone.'
+                msg.format(CONF.enforcement.project_enforcement_id)
+                LOG.info(msg)
+                raise ValueError(msg)
```

Comment:
I think when you raise a ValueError here, it won't be caught in the `except` block below. Was this intentional?

Reply:
Yes, the except block was meant to handle a different exception. I just wanted the ValueError to log the issue.